simulation/test_dht.cpp

```diff
@@ -143,7 +143,7 @@ TORRENT_TEST(dht_bootstrap)
 				ses.post_session_stats();
 				std::printf("depth: %d nodes: %d\n", routing_table_depth, num_nodes);
 				TEST_CHECK(routing_table_depth >= 8);
-				TEST_CHECK(num_nodes >= 110);
+				TEST_CHECK(num_nodes >= 50);
```

Comment:
that makes sense. it may still be a (separate) problem that the bootstrap perhaps focuses too much on depth and not so much on breadth.